src/server/models/cacheable_queries/opt-out.js

```diff
@@ -1,12 +1,13 @@
 import { r, OptOut } from '../../models'
 
 // STRUCTURE
-// maybe HASH by organization, so optout-<organization_id> has a <cell> key
+// SET by organization, so optout-<organization_id> has a <cell> key
+//  and membership can be tested
 
 const orgCacheKey = (orgId) => (
 !!process.env.OPTOUTS_SHARE_ALL_ORGS
- ? `${process.env.CACHE_PREFIX | ''}optouts`
- : `${process.env.CACHE_PREFIX | ''}optouts-${orgId}`)
+ ? `${process.env.CACHE_PREFIX || ''}optouts`
```

Comment:
were these `|`s all meant to be ORs before and haven't been working?

Reply:
exactly.  These were accidental bitOR but should have been ORs.

Reply:
exactly.  These were accidental bitOR but should have been ORs.